habitat/plan.sh

```diff
@@ -15,7 +15,7 @@ pkg_deps=(
 pkg_build_deps=(
     core/coreutils
     core/git
-    core/yarn
+    idolgirev/yarn
```

Comment:
this plan has an updated version of yarn (which I *think* helps speed up build times). I tried to build our own but failed. This can be reverted without problem if desired.